bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_rtensor.py

```diff
@@ -114,6 +113,25 @@ def RTensorGetitem(self, idx):
         idxVec[i] = x
     return self(idxVec)
 
+def RTensorInit(self, *args):
+
+    if (len(args) == 1) :
+        try:
+            import numpy as np
+        except:
+            print("Failed to import numpy in RTensor constructor")
+        if isinstance(args[0], np.ndarray):
+            data = args[0]
+            #conversion from numpy to buffer float/double * works only if C order
+            if (data.flags.c_contiguous):
+                shape = data.shape
+                from cppyy.gbl import TMVA
+                layout = TMVA.Experimental.MemoryLayout.RowMajor
+                return self._original_init_(data,shape,layout)
+            else:
+                print("Error- cannot convert a Numpy array in Fortran Format to a C array")
```

Comment:
With this `if-else` statement, we are just printing an error message but not actually raising an error. I don't think this is the behaviour we want right? If the user is trying to convert a numpy array to RTensor and something goes wrong, the application should stop. Consider changing the `else` block to
```python
else:
    raise ValueError("Can only convert C-contiguous Numpy arrays to RTensor but input array is Fortran-contiguous")
```